introduction_to_time_series_analysis_in_r.Rmd

```diff
@@ -0,0 +1,195 @@
+# Introduction to Time Series Analysis in R
+
+Chenqi Wang
+
+```{r, include=FALSE}
+# this prevents package loading message from appearing in the rendered html file
+knitr::opts_chunk$set(warning = FALSE, message = FALSE)
+
+```
+
+### 1. Overview
```

Comment:
Subheading should not contain numbers. They'll be automatically created when rendering.